data_broker.js

```diff
@@ -222,9 +222,29 @@ class DataBroker extends EventTarget {
   }
 
   async _updateStats() {
-    const validRegions = this.regions ? this.regions : getValidRefs(this._header, this._readDepthData);
+    const validBamHeader = getValidRefs(this._header, this._readDepthData);
```

Comment:
Sorry, I know I jumped around last night while we were talking about this, but I don't think we should put this in the data broker. I think a better design would be to have an event listener in index.html that listens for the header/readDepthData, then if the mapped reads data is available it calls mappedReadsChart.update() with the correct values.

This has the following advantages:

* We can put the data setting code and the color setting code in the same place
* Don't have to modify the data broker
* Don't have to modify the percent chart